js/lib/src/instructions/namespace.ts

```diff
@@ -85,6 +88,7 @@ export interface UncacheArtifactArgs {
 export interface UncacheArtifactAccounts {
   namespaceMint: web3.PublicKey;
   artifact: web3.PublicKey;
+  raindropsProgram: web3.PublicKey;
```

Comment:
Make this an enum of the various raindrops program account pub keys

Reply:
Wasn't able to get this to work due to the PublicKey type, would like a suggestion

Reply:
You can use a numeric or string based enum and have a namespace that contains a helper to convert it to the actual PubKey. 

https://github.com/raindrop-studios/rain-redemptions/blob/0ea1720fa9ec979516e85eefbb969024aa57e4b0/app/lib/src/state/redemption.ts#L23